server/jetty-app/README.md

```diff
@@ -1,70 +1,29 @@
 # server-jetty-app
 
-## Native development
-
 This document is oriented towards getting a server up for local development.
-If you are interested in running a native production release, please see
+If you are interested in running a native production release or to see other options for starting the server, please see
 [https://deephaven.io/core/docs/how-to-guides/configuration/native-application/](https://deephaven.io/core/docs/how-to-guides/configuration/native-application/).
 
-### Groovy Quickstart
-
-```shell
-./gradlew server-jetty-app:run -Pgroovy 
-```
-
-### Python Quickstart
-
-1. Setup virtual environment:
-
-   ```shell
-   python -m venv /tmp/my-dh-venv
-   source /tmp/my-dh-venv/bin/activate
-   ```
-
-1. Build and install wheel
-
-   ```shell
-   ./gradlew py-server:assemble
+If using Python, see [Python development guide](../../py/README.md) for setting up the Python environment.
```

Comment:
Someone coming to this is going to be totally confused.  Here Python devs are directed away, but below, it shows how to run python.